renovate.json

```diff
@@ -2,13 +2,37 @@
   "commitMessageAction": "update",
   "commitMessagePrefix": "⬆️(dependencies)",
   "commitMessageTopic": "{{depName}}",
-  "enabledManagers": ["npm"],
-  "extends": ["config:base"],
+  "enabledManagers": [
+    "npm",
+    "setup-cfg"
+  ],
+  "extends": [
+    "config:base"
+  ],
+  "labels": [
+    "dependencies"
+  ],
   "packageRules": [
     {
-      "packagePatterns": ["*"],
+      "matchManagers": [
+        "npm"
+      ],
       "groupName": "npm dependencies",
-      "schedule": ["before 7am on monday"]
+      "schedule": [
+        "before 7am on monday"
+      ]
+    },
+    {
+      "matchManagers": [
+        "setup-cfg"
+      ],
+      "matchDepTypes": [
+        "devDependencies"
```

Comment:
If I understand the [documentation](https://docs.renovatebot.com/configuration-options/#matchdeptypes), you limit the packages to `devDependencies`. But in `setup.cfg` this is not existing and why limiting to development ?
This part is unclear for me.

Reply:
I limited to development because previously pyup ignored `install_requires` dependency (except django-redis and exrex don't no why just this deps are allowed).

Then about the use of `devDependencies`, I don't know yet which value to use (`options.extras_require`?).
Or maybe I should use [`excludePackageNames`](https://docs.renovatebot.com/configuration-options/#excludepackagenames) and declare one by one each dependency to ignore in this array?

Reply:
Ok I didn't notice in the PR the removal of `pyup` comments, sorry. Did you try to use the `renovate-config-validator` program ? https://docs.renovatebot.com/reconfigure-renovate/

Reply:
Yes, I did it.

Reply:
So to know if it works we have to merge it and wait to see ?

Reply:
This scenario looks good :+1: